src/data_model/transformations.jl

```diff
@@ -47,3 +47,19 @@ function apply_voltage_bounds!(data_eng::Dict{String,<:Any}; vm_lb::Union{Real,M
         end
     end
 end
+
+
+"removes all fields ending in '_ub' or '_lb'"
+function rm_bounds!(data_eng)
+    for (k,v) in data_eng
```

Comment:
I suggest renaming this function to be more verbose, like `remove_all_bounds!`; I think we will want different variants of transformation functions that remove bounds, therefore keeping the names clear will be extra important